docs/extensibility/display-widgets.md

```diff
@@ -125,6 +125,27 @@ Table widgets display array data as rows of a table instead of free-standing com
 
 <img src="./assets/display-widgets/Table.png" alt="Example of a table widget" style="border: 1px solid #D2D5D9">
 
+### ResourceList
+
+ResourceList widgets render a list of Kubernetes resources. It should be used along with [related resources](resources.md#relations-section).
```

Comment:
What "it"? The only singular word in the previous sentence is "a list", so is that what should be used along with related resources? 

For clarification, I suggest to replace "it" with the actual word, not sure which one it is:

```suggestion
ResourceList widgets render a list of Kubernetes resources. The resource list should be used along with [related resources](resources.md#relations-section).
```
or 

```suggestion
ResourceList widgets render a list of Kubernetes resources. The ResourceList widgets should be used along with [related resources](resources.md#relations-section).
```

```suggestion
ResourceList widgets render a list of Kubernetes resources. The Kubernetes resources should be used along with [related resources](resources.md#relations-section).
```